include/picongpu/fields/background/templates/twtstight/BField.tpp

```diff
@@ -205,29 +205,13 @@ namespace picongpu
                 auto const beta0 = float_T(beta_0);
```

Comment:
Is there a particular benefit in using your own float here? I see it is currently `float_X` and was previously `float_64`. Is this a remainder that you want to keep?

Reply:
I would like to keep this, as I need to be able to switch between all float_64 computations of twtstight and the faster float_X /float_X hybrid implementation.